Models/ChangelogManager.php

```diff
@@ -48,6 +48,11 @@ private function loadChangelog(): void
         $curl = curl_init();
         curl_setopt($curl, CURLOPT_URL, self::GITHUB_API_RELEASES_URL.self::RELEASE_IDS[self::LATEST_VERSION]);
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+
+        // fix chyby s SSL certifikátem - pouze pro vývoj stránek
```

Comment:
TODO Přidat do index.php nějakou konstantu `PRODUCTION_MODE`, která toto nastavení povolí pouze při nastavení na `false`. To samé i pro věci kolem kontroly SSL a e-mailů.

Reply:
Spraveno, viz #272